patterns/sailfish-api-3.yaml

```diff
@@ -0,0 +1,30 @@
+Summary: Sailfish API Level 3
+Description: Pre-installed Qt plugins and packages for allowed Harbour APIs.
+Name: sailfish-api-3
+Provides:
+    - sailfish-api = 3
+Requires:
+# Begin requirements inserted by update.py
+    - qt5-qtdeclarative-import-xmllistmodel
+    - qt5-qtdeclarative-import-folderlistmodel
+    - qt5-qtdeclarative-import-localstorageplugin
+    - qt5-qtdeclarative-import-multimedia
+    - qt5-qtqml-import-webkitplugin
+    - qt5-qtdeclarative-import-particles2
+    - qt5-qtdeclarative-qtquickparticles
+    - qt5-qtsvg
+    - qt5-qtgraphicaleffects
+    - qt5-qtdeclarative-import-positioning
+    - qt5-qtdeclarative-import-sensors
+    - qt5-qtquickcontrols-layouts
+    - qt5-qtdeclarative-import-models2
+    - qt5-qtmultimedia
+    - qt5-qtmultimedia-plugin-audio-pulseaudio
+    - qt5-qtpositioning
+    - qt5-plugin-imageformat-gif
+    - qt5-plugin-imageformat-ico
+    - qt5-plugin-imageformat-jpeg
+    - qt5-qtsvg-plugin-imageformat-svg
+    - libsailfishapp
+    - SDL2
```

Comment:
Most subpackages are examples or debug libraries, I read them through earlier. But I'll cross-check with harbour white list again, I may be missing something.